string 				shape  # 'constant' or 'ramp' or 'circle' or 'square' or 'flylogo' or 'spiral'
float64 			hzPattern
float64 			hzPoint
int32 				count  # -1 means forever
geometry_msgs/Point size
float64             param

